action.yaml

```diff
@@ -65,8 +65,9 @@ inputs:
     required: false
     default: ''
   hide-progress:
-    description: 'hide progress output'
+    description: 'suppress progress bar and log output'
     required: false
+    default: 'true'
```

Comment:
This shouldn't be turned on by default as users expect to the see the output.

Reply:
Thanks for clearing this up, as there were discrepancies between the doc and the code.
Updated.